src/app/backend/integration/api/types.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// IntegrationID is a unique identification string that every integrated app has to provide.
```

Comment:
We could keep all of them as const values in this file. WDYT?

I don't think we have to worry about name collisions here, so simple integration name should be sufficient for the foreseeable future.